frontend/packages/ceph-storage-plugin/src/utils/common.ts

```diff
@@ -1,7 +1,13 @@
+import { TFunction } from 'i18next';
 import { StorageClusterKind } from '../types';
 
 export const checkArbiterCluster = (storageCluster: StorageClusterKind): boolean =>
   storageCluster?.spec?.arbiter?.enable;
 
 export const checkFlexibleScaling = (storageCluster: StorageClusterKind): boolean =>
   storageCluster?.spec?.flexibleScaling;
+
+export const commaSeparatedString = (text: string[], t: TFunction): string =>
+  [text.slice(0, -1).join(', '), text.slice(-1)[0]].join(
+    text.length < 2 ? '' : ` ${t('ceph-storage-plugin~and ')}`,
```

Comment:
i uded ` ${t('ceph-storage-plugin~and ')}` is working

Reply:
For translation, its better to not put any spaces with strings. Also there is a chance of accidental miss while modifying.
Also explicit indication of spaces misses the chance of removal while making any changes.
```suggestion
    text.length < 2 ? '' : ` ${t('ceph-storage-plugin~and')} `,
```